### PR TITLE
Made the RIAA DMCA actually say what it is.

### PR DESCRIPTION
**Pull request by Sven65**

### CODE REVIEW

**Review by JaxTheWolf — APPROVED** [review]
Spot on edit.

**Review by gliese1337 — APPROVED** [review]

**Review by Jeremiidesu — APPROVED** [review]

**Comment by OkameProductions** [comment]
beautiful

**Review by imachug — APPROVED** [review]
Way better now.

**Review by Pokerkoffer — APPROVED** [review]

**Review by Suyashtnt — APPROVED** [review]
yes

**Review by lucario — APPROVED** [review]

**Comment by dfault-user** [comment]
Amazing

**Review by kakolisgay — APPROVED** [review]

**Comment by GingkathFox** [comment]
was giggling the entire time, you get my vote 👍

**Review by guesant — APPROVED** [review]

**Comment by Gustavo6046** [comment]
Accuracy.

**Review by btwiusegentoo — APPROVED** [review]
Looks good to me

**Review by guesant — APPROVED** [review]
:+1:

**Review by Gustavo6046 — APPROVED** [review]
This is a blatant way to put what this takedown really means. Yet it's also so painfully true. Raw unrefined words hurt the most. (Although you're not supposed to use them any time of day in any occasion! They can have pretty drastic effects.)

I love this.

**Review by Karitham — APPROVED** [review]

**Review by GingkathFox — APPROVED** [review]

File: 2020/10/2020-10-23-RIAA.md
Comment (MineRobber9000) [inline 2020/10/2020-10-23-RIAA.md:17]: Hahaha! Well put!

**Review by MineRobber9000 — APPROVED** [review]

File: 2020/10/2020-10-23-RIAA.md
Comment (turtiustrek) [inline 2020/10/2020-10-23-RIAA.md:30]: this is too nice....

**Comment by mininmobile** [comment]
image

**Review by mininmobile — APPROVED** [review]

**Review by DominykasPetke — APPROVED** [review]

File: 2020/10/2020-10-23-RIAA.md
Comment (Shijikori) [inline 2020/10/2020-10-23-RIAA.md:11]: They do-ish. A lot of internet services are based in USA and so under american law... which sucks but i agree that it feels like an abuse of the law to take down lawful code and software

**Review by Razuatix — APPROVED** [review]

**Review by pro7357 — APPROVED** [review]

**Review by iczero — APPROVED** [review]

**Comment by iczero** [comment]
Oh look, a leaked internal memo.

**Review by lululombard — APPROVED** [review]

**Review by AliceDiNunno — APPROVED** [review]

**Review by Jeder321 — APPROVED** [review]

**Review by L33Tech — APPROVED** [review]
yep

**Review by MatteoGheza — APPROVED** [review]

**Comment by henry701** [comment]
@github might want to merge this, he made the source more terser and readable, conveying intent. Good Code Quality edit!

File: 2020/10/2020-10-23-RIAA.md
Comment (ValZapod) [reply 2020/10/2020-10-23-RIAA.md:11]: So what? If they will sue and the guy (imagine in Russia) will show up in court, it will lack jurisdiction and dismissed with prejudice.

**Review by Shijikori — APPROVED** [review]

**Review by Joshua-Ashton — APPROVED** [review]

**Review by uis246 — APPROVED** [review]
I agree with you

**Review by bossoff — APPROVED** [review]

**Review by idkwuu — APPROVED** [review]

**Review by luqmansen — APPROVED** [review]

File: 2020/10/2020-10-23-RIAA.md
Comment (wallabra) [reply 2020/10/2020-10-23-RIAA.md:17]: What if it's intentional? Maybe the RIAA loved youtube-dl and want everyone to find it via reverse psychology! :o

**Review by advilm — APPROVED** [review]

**Review by dicedtomatoreal — APPROVED** [review]

**Review by Kahdeg-15520487 — APPROVED** [review]

**Review by illusion0001 — APPROVED** [review]

**Review by williamd1k0 — APPROVED** [review]

**Review by dev-natalie — APPROVED** [review]

File: 2020/10/2020-10-23-RIAA.md
Comment (Merith-TK) [inline 2020/10/2020-10-23-RIAA.md:23]: "We ***THINK*** that this activity is not authorized by the copyright owner, its agent, or the law. We assert that the information in this notification is accurate, based upon the data available to us. 
We have not actually asked the copyright owner, we think therefore it must be true"

**Review by bkhl — APPROVED** [review]

**Review by jso8910 — APPROVED** [review]

File: 2020/10/2020-10-23-RIAA.md
Comment (ghost) [reply 2020/10/2020-10-23-RIAA.md:11]: @ValZapod it's cute that you think that. hey remember all the times the cia assassinated foreign leaders so the us could create puppet states?

the us doesn't care about jurisdiction, and will enforce its laws as it pleases

**Review by alexis-evelyn — APPROVED** [review]

**Review by Keanu73 — APPROVED** [review]

**Review by stanislavkozlovski — APPROVED** [review]

**Review by JordyEGNL — APPROVED** [review]
Good edit! Good job.

**Review by analotia — APPROVED** [review]

**Comment by abhitrueprogrammer** [comment]
Lets hope now they gonna take this dmca request down, cause it would hurt their repo sooo bad.

**Review by abhitrueprogrammer — APPROVED** [review]
I don't care how much time it took you to create this change, but it just HAS to be commited right now.

**Comment by Jeder321** [comment]
this is a joke if you didn't know
________________________________

**Review by 06kellyjac — APPROVED** [review]

**Review by Techwolf12 — APPROVED** [review]

**Comment by BlueRaccoonTech** [comment]
Hm? I'm confused as to what this pull request does, this looks exactly the same as the original text.

**Review by CrystalVulpine — APPROVED** [review]

**Review by JeDaYoshi — APPROVED** [review]

**Review by dfault-user — APPROVED** [review]

**Review by yannicklamprecht — APPROVED** [review]

**Review by InfinityGhost — APPROVED** [review]

**Review by ZeProf2Code — APPROVED** [review]

**Review by kubapolish — APPROVED** [review]

File: 2020/10/2020-10-23-RIAA.md
Comment (SrGaabriel) [inline 2020/10/2020-10-23-RIAA.md:28]:
```suggestion
Fuck you, oh wait, you were already fucked by our monopoly.
```

**Review by SrGaabriel — APPROVED** [review]

File: 2020/10/2020-10-23-RIAA.md
Comment (saiintbrisson) [inline 2020/10/2020-10-23-RIAA.md:7]: I suggest the following: "Here at RIAIAIAIA, we put our time and effort into gently fucking people's lives and freedom, not even our artists have rights over their own songs"

**Review by SaiintBrisson — CHANGES_REQUESTED** [review]

**Review by LorenzooG — APPROVED** [review]
Yes

**Review by HenryFabio — APPROVED** [review]
Mama aqui

**Review by Nightdavisao — APPROVED** [review]
gud

File: 2020/10/2020-10-23-RIAA.md
Comment (whleucka) [reply 2020/10/2020-10-23-RIAA.md:7]: Nice

**Review by zkingboos — APPROVED** [review]
👍 RIAA S*CKS

**Review by sarathyweb — APPROVED** [review]

**Review by xf8b — APPROVED** [review]
iapprove

**Comment by MrMelon54** [comment]
Why don't the devs of youtube-dl just run a gitbucket or bitbucket server on their website and host the source from there? Github obviously doesn't care about their hard work making a useful tool that so make people use and deleting the Github repo won't stop them and won't stop people downloading it from other places so it's absolutely useless to delete the repo anyway.

**Review by MrMelon54 — APPROVED** [review]

**Review by BitesizedLion — APPROVED** [review]

**Review by PieterjanDeClippel — APPROVED** [review]
MERGE RIGHT NOW !!!

**Comment by 3n-k1** [comment]
@MrMelon54 we all know the takedown is illegal, but do you think the devs have the money, time, and energy to fight it? most people don't

**Comment by MrMelon54** [comment]
One question I really want answered is "why didn't Github fight it?"

**Review by simon511000 — APPROVED** [review]

**Comment by 3n-k1** [comment]
github didn't fight it because they don't care about users, plain and simple. in a capitalist system, money is more important than people

**Comment by MrMelon54** [comment]
that sounds like pretty much all major companies nowadays

**Review by DaWe35 — APPROVED** [review]

**Comment by interbiznw** [comment]
> Why don't the devs of youtube-dl just run a gitbucket or bitbucket server on their website and host the source from there? Github obviously doesn't care about their hard work making a useful tool that so make people use and deleting the Github repo won't stop them and won't stop people downloading it from other places so it's absolutely useless to delete the repo anyway.

Anyone hosting the files will be subject to the same issues they need to host their own gitlab on their own server, or put the repo on i2p, good luck taking that down. And yes repos can be hosted and fully edited as they are on github from i2p

**Review by Nefomemes — APPROVED** [review]

**Comment by moorecode** [comment]
Correct:

image

**Review by nixklai — APPROVED** [review]

File: 2020/10/2020-10-23-RIAA.md
Comment (interbiznw) [reply 2020/10/2020-10-23-RIAA.md:17]: Well they definitely got that Streisand effect going good now hahahahah

**Review by Amunak — APPROVED** [review]

**Comment by DaWe35** [comment]
Without youtube-dl I'm unable to restream MY OWN live streams. There are many projects what are 100% legal, what the fuck are you doing @github

**Comment by Hans5958** [comment]
Ladies and gentlemen, may I have your attention on this court order added hours ago?

https://github.com/github/dmca/blob/master/2020/10/2020-10-23-RIAA-court-order.pdf

**Comment by MrMelon54** [comment]
Can someone translate that to English as I don't speak lawyer

**Comment by Hans5958** [comment]
it's called "legalese"

**Comment by MrMelon54** [comment]
It doesn't matter what its called I'm just saying I don't understand any of it. I'm assuming this is why people pay lawyers to write letters 🤣

**Comment by Hans5958** [comment]
yeah i get it

**Comment by alexis-evelyn** [comment]
`S-Variable`. Please, and still with the "rolling cipher"...

**Review by CriticalElement — APPROVED** [review]

**Review by CyberTailor — APPROVED** [review]
Submit review

**Comment by jessephus** [comment]
Hi :wave: We are closing this pull request because we [do not accept contributions](https://github.com/github/dmca/blob/master/CONTRIBUTING.md) to this repository. Feel free to keep the excellent discussion going in this thread. Please just be mindful to keep the discussion respectful and consistent with [GitHub’s Community Guidelines](https://docs.github.com/github/site-policy/github-community-guidelines).

We understand that you may have questions about this takedown. Although we do not normally monitor comments posted to this repository, our team is reading through the comments and questions in this and other recent threads to see where we may be able to clarify our policies or answer some common questions. Thanks.

**Comment by GingkathFox** [comment]
oh wow, theyre actually reading the comments here?

**Comment by Hans5958** [comment]
> oh wow, theyre actually reading the comments here?

They are closing PRs left and right, but that doesn't mean you can't discuss it anymore.

**Comment by Joshua-Ashton** [comment]
@jessephus, how can you consider closing this PR when it has so many positive approvals?

Surely it must have some merit...

Maybe it would help if we got an expert like @kassindornelles to review, it could get some traction?

**Comment by Hans5958** [comment]
UPDATE: It's official, RIAA just got their L. youtube-dl is back, boys!

GitHub's statement:
- https://github.blog/2020-11-16-standing-up-for-developers-youtube-dl-is-back/
- https://twitter.com/GitHubPolicy/status/1328365918750003203
- https://twitter.com/natfriedman/status/1328365679473426432

youtube-dl's statement: ytdl-org/youtube-dl#27013

DMCA reversal letter:
- https://github.com/github/dmca/blob/master/2020/11/2020-11-16-RIAA-reversal-effletter.pdf
- https://github.com/github/dmca/commit/e00bfb544e93bfd3066fe1699171964dd2dc29e0

(for those who aren't aware and future readers)

**Comment by pedro2555** [comment]
lol

**Comment by simon511000** [comment]
xD

**Comment by GingkathFox** [comment]
YEAAAAAAAH BOIIIIIIIII

**Comment by Sven65** [comment]
We did it, fuck the RIAA!

**Comment by PieterjanDeClippel** [comment]
Long live open-source !!!

**Pull request closed without merging**
